lib/FusionInventory/Agent/Task/Inventory/Win32/Storages/HP.pm

```diff
@@ -0,0 +1,55 @@
+package FusionInventory::Agent::Task::Inventory::Win32::Storages::HP;
+
+use strict;
+use warnings;
+
+use parent 'FusionInventory::Agent::Task::Inventory::Module';
+
+use English qw(-no_match_vars);
+use UNIVERSAL::require;
+
+use FusionInventory::Agent::Tools::Storages::HP;
+
+sub _getHpacuacliFromWinRegistry {
```

Comment:
Better move this private function at the end of the file as I prefer to see isEnabled() and doInventory() before the rest.